pkg/api/helpers.go

```diff
@@ -79,19 +79,42 @@ var Semantic = conversion.EqualitiesOrDie(
 	},
 )
 
-var standardResources = sets.NewString(
-	string(ResourceMemory),
+var standardComputeResources = sets.NewString(
```

Comment:
Splitting based on integer values seems like a simple cleanup.  Will update and poke.  Thanks!